recidiviz/ingest/us_co_mesa/us_co_mesa_scraper.py

```
@@ -0,0 +1,196 @@
+        elif self.should_get_more_tasks(task_type):
+            params_list.extend(self._get_person_params(content))
+
+        if self.should_scrape_person(task_type):
+            self._person = UsCoMesaScraper._scrape_person(content)
+        if self.should_scrape_record(task_type):
+            self._record = UsCoMesaScraper._scraper_record(content)
+
+        return params_list
+
+    def _get_all_people_params(self):
+        params = {
+            'endpoint': self._initial_endpoint,
+            'task_type': constants.GET_MORE_TASKS,
+            'data': {
+                'SearchField': 'LName',
+                'SearchVal': '',
+            },
+        }
+        return params
+
+    def _get_person_params(self, content):
+        params_list = []
+
+        form_inputs = content.cssselect('[name=SearchVal]')
+        for form_input in form_inputs:
+            params_list.append({
+                'endpoint': self._initial_endpoint,
+                'task_type': constants.SCRAPE_PERSON_AND_RECORD_AND_MORE,
+                'data': {
+                    'SearchField': 'BookingNo',
+                    'SearchVal': form_input.get('value')
+                },
+            })
+
+        return params_list
+
+    # TODO(colincadams): Parse free form text in first table row, current
+    # possibilities:
+    # - 'Inmate has been sentenced and cannot post bond for release at this
+    #    time.'
+    # - 'The inmate currently has open charges. Inmate may have a hold or bond
+    #    has not been set at this time...'
+    # - 'The inmate currently is eligible for release after posting bond...'
+    # - 'The inmate currently has a hold that could prevent him from being
+    #    eligible for release after posting bond...'
+
+    @staticmethod
+    def _scrape_person(content):
+        person = Person()
+
+        table = content.cssselect('table')[0]
+        for row in table:
+            if len(row) == 2:
```

If the row has only one element then the value we want to extract is blank? I'd add an inline comment on that, since `if length is 2` otherwise appears magical.

---

I've actually just deleted this line so that we don't silently skip data if the page format change (i.e. all rows are changed to have 3 values), this is inline with my next comment. I do think that this is a part of a bigger assumption about how we handle page format changes and a potential abundance of KeyErrors :)